README.md

```diff
@@ -333,7 +333,7 @@ below the line `.package(url:
 
 4. Change the target for Application to include SwiftKueryORM and SwiftKueryPostgreSQL after Health
 ```swift
-.target(name: "Application", dependencies: [ "Kitura", "Configuration", "CloudEnvironment","SwiftMetrics","Health", "SwiftKueryORM", "SwiftKueryPostgreSQL"]),
+.target(name: "Application", dependencies: [ "Kitura","CloudEnvironment","SwiftMetrics","Health", "SwiftKueryORM", "SwiftKueryPostgreSQL"]),
```

Comment:
I wonder if it's better to put the new entries at the start of the array. That way they are always visible without scrolling right.

Reply:
Could go either way. Reading left to right encourages thinking about Kitura "first" but it truly doesn't matter that much. Will leave as-is unless someone else speaks up.